### PR TITLE
Rename TerminatedRoundArtefacts to PreparedRoundArtefacts

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/RoundSpecificPeers.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 import tech.pegasys.pantheon.consensus.ibft.payload.PreparePayload;
 import tech.pegasys.pantheon.consensus.ibft.payload.RoundChangePayload;
 import tech.pegasys.pantheon.consensus.ibft.payload.SignedData;
-import tech.pegasys.pantheon.consensus.ibft.statemachine.TerminatedRoundArtefacts;
+import tech.pegasys.pantheon.consensus.ibft.statemachine.PreparedRoundArtefacts;
```
Comment (jframe) [inline consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/RoundSpecificPeers.java]: I thought we were going to use the American spelling?

Comment (rain-on) [reply consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/RoundSpecificPeers.java]: We will - but want to do that in a secondary PR.

Comment (jframe) [reply consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/RoundSpecificPeers.java]: fair enough, that works

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java [hunk 2]
```diff
@@ -133,20 +133,20 @@ public void roundExpired(final RoundExpiry expire) {
...
         "Round has expired, creating PreparedCertificate and notifying peers. round={}",
         currentRound.getRoundIdentifier());
-    final Optional<TerminatedRoundArtefacts> terminatedRoundArtefats =
-        currentRound.constructTerminatedRoundArtefacts();
+    final Optional<PreparedRoundArtefacts> preparedRoundArtefats =
+        currentRound.constructPreparedRoundArtefacts();
 
-    if (terminatedRoundArtefats.isPresent()) {
-      latesteTerminatedRoundArtefacts = terminatedRoundArtefats;
+    if (preparedRoundArtefats.isPresent()) {
```
Comment (jframe) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: misspelling of Artefacts

File: consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManagerTest.java [hunk 3]
```diff
@@ -152,10 +152,10 @@ private RoundChange makeRoundChangeMessageWithPreparedCert(
                 })
             .collect(Collectors.toList());
 
-    final TerminatedRoundArtefacts terminatedRoundArtects =
-        new TerminatedRoundArtefacts(proposal, preparePayloads);
+    final PreparedRoundArtefacts preparedRoundArtects =
```
Comment (jframe) [inline consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManagerTest.java]: misspelling of Artefacts

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java (hunk 2 above)
Comment (rain-on) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftBlockHeightManager.java]: done.

File: consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManagerTest.java (hunk 3 above)
Comment (rain-on) [reply consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManagerTest.java]: done.

**Pull request merged**